test/functional/testlib.bash

```diff
@@ -861,6 +862,8 @@ add_to_manifest() { # swupd_function
 		          updates are to be done in the manifest to avoid extra processing
 		    -e    If set (experimental), the bundle will be added to the manifest as
 		          experimental. This should only be used when <manifest> is a MoM
+		    -l    If set (do not follow links), if a link is being added, the file
```

Comment:
Instead of adding this parameter, I think that function add_to_manifest() should never include the file that is being liked in the manifest. If the user wants to include it, the user can do that explicitly. That's how most tool works and how swupd usually handle files. For example:
```
touch a
ln -s a b
tar cjf file.tar b # a is not automatically included.
```

Reply:
Another win on that is that we don't need a recursion in add_to_manifest(). I have the feeling that this function has an infinite loop if you have a link pointing to itself.

Reply:
looks like we were already explicitly adding those files anyway, so making this change shouldn't be a problem. Good eye.